librus/__init__.py

```diff
@@ -178,7 +178,7 @@ def list_messages(self, get_content=False):
         """
         response = self._html_session.get(url='https://synergia.librus.pl/wiadomosci')
 
-        for row in response.html.find('.line0') + response.html.find('.line1'):
+        for row in response.html.find('.stretch > tbody > tr'):
```

Comment:
intentional? (the commit message says `Setting message content directly`, so i want to make sure)

Reply:
Yep, it is all right. Just a quick fix to get the messages in order they appear in the html.